packages/datatrak-web/src/views/LandingPage/ActivityFeedSection/ActivityFeedMarkdownItem.tsx

```diff
@@ -40,8 +41,8 @@ const Header = styled.div<{
 `;
 
 const Logo = styled.img.attrs({
-  src: '/bes-logo.png',
-  alt: 'BES logo',
+  'aria-hidden': true,
```

Comment:
![Screenshot 2025-02-10 at 5 07 32 PM](https://github.com/user-attachments/assets/096c1c37-f240-4f10-8a6e-06a5737c13b6)

I've been getting this error since we added aria-hidden else where. I think it might be applicable to this use case too since the logo is focusable.